cloudformation/media-atom-maker.json

```diff
@@ -365,7 +370,7 @@
                                 {
                                     "Effect": "Allow",
                                     "Action": ["sts:AssumeRole"],
-                                    "Resource": [{ "Ref": "ContentAtomArn" }, { "Fn::GetAtt" : ["LimitedUploadRole", "Arn"] }]
```

Comment:
it is, but I've extracted into a new `CrossAccountAccessPolicy` resource that then gets applies to the relevant roles as its a little DRYer